src/storage/hummock_sdk/src/compact.rs

```diff
@@ -78,18 +96,22 @@ pub fn append_sstable_info_to_string(s: &mut String, sstable_info: &SstableInfo)
         hex::encode(key_range.right.as_slice())
     };
 
-    let ratio = sstable_info.stale_key_count * 100 / sstable_info.total_key_count;
+    let stale_ratio = sstable_info.stale_key_count * 100 / sstable_info.total_key_count;
```

Comment:
total_key_count be 0?

Reply:
good catch, 

It is not expected that there will be an sst with total_key_count = 0, however,  I can't guarantee that the arguments passed by the caller are compliant.